packages/astro/src/runtime/server/index.ts

```diff
@@ -11,6 +11,10 @@ export { createMetadata } from './metadata.js';
 export { escapeHTML, unescapeHTML } from './escape.js';
 
 const voidElementNames = /^(area|base|br|col|command|embed|hr|img|input|keygen|link|meta|param|source|track|wbr)$/i;
+const htmlBooleanAttributes = /^(allowfullscreen|async|autofocus|autoplay|controls|default|defer|disabled|disablepictureinpicture|disableremoteplayback|formnovalidate|hidden|loop|nomodule|novalidate|open|playsinline|readonly|required|reversed|scoped|seamless|itemscope)$/i;
+const htmlEnumAttributes = /^(contenteditable|draggable|spellcheck|value)$/i;
+// Note: SVG is case-sensitive!
+const svgEnumAttributes = /^(autoReverse|externalResourcesRequired|focusable|preserveAlpha)$/;
```

Comment:
After further investigation:

> All attribute names are case-insensitive, including attributes on SVG and MathML elements.
> — https://htmlparser.info/syntax/#attributes

I was able to test this using https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/preserveAlpha. I tested other, more common case-sensitive attributes as well: 
```html
<svg viewbox="0 0 32 32" width="16" height="16">
	<rect x="0" y="0" width="16" height="16" rx="4" />
</svg>
```

```suggestion
const svgEnumAttributes = /^(autoReverse|externalResourcesRequired|focusable|preserveAlpha)$/i;
```